src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SuggestionModeCompletionProviderTests.cs

```diff
@@ -472,7 +472,7 @@ class Program
 {
     static void Main(string[] args)
     {
-        var x = (a: a, b$$)
+        System.Action x = (a: a, b$$)
```

Comment:
Can you add a new test that tests the "var" case so we can keep track of all these behaviors?  Thanks!